fs/src/commonTest/kotlin/com/suparnatural/core/fs/test.kt

```diff
@@ -132,15 +111,16 @@ class FileSystemTests {
         assertTrue(FileSystem.writeFile(testFilePath, "", false, ContentEncoding.Utf8))
         assertTrue(FileSystem.appendFile(testFilePath, testString, false, ContentEncoding.Base64))
         assertTrue(FileSystem.appendFile(testFilePath, testString2, false, ContentEncoding.Base64))
-        assertEquals("$testJoinedBase64String", FileSystem.readFile(testFilePath, ContentEncoding.Utf8))
+        assertEquals(testJoinedBase64String, FileSystem.readFile(testFilePath, ContentEncoding.Utf8))
     }
 
 
     @Test
     fun testMkDir() {
         val paths = listOf(Triple("dir", second = false, third = true), Triple("dir1/dir2", second = true, third = true), Triple("dir4/dir5", second = false, third = false))
         paths.forEach {
-            val path = testPath.absolutePath?.byAppending(it.first)!!
+            val path = testDirectory.absolutePath?.byAppending(it.first)!!
+            println(path)
```

Comment:
Remove?

Reply:
Thanks, forgot about it. Remove it with the latest commit